import.py

```diff
@@ -243,9 +243,12 @@ def modelClassGen(stem, sf, keyField, srs, shapeType):
 
 
 def modelsGeoFilterGen(stem, keyField):
-  text = "        qs_" + stem + " = " + stem + ".objects.filter(geom__contains=pnt)\n"
+  text  = "        qs_" + stem + " = " + stem + ".objects.filter(geom__contains=pnt)\n"
```

Comment:
I wouldn't normally complain about an extra space but I think it makes a difference here.